client/my-sites/site-settings/test/form-general.jsx

```diff
@@ -35,28 +50,50 @@ moment.tz = {
 	guess: () => moment(),
 };
 
+const initialReduxState = {
```

Comment:
This is a good opportunity for a follow-up PR that moves this function to a separate helper module